dev/com.ibm.ws.security.acme/resources/com/ibm/ws/security/acme/resources/AcmeMessages.nlsprops

```diff
@@ -10,19 +10,156 @@
+CWPKI2017E.explanation=Requested the terms of service from the certificate authority, but an exception occurred.
+CWPKI2017E.useraction=Review the provided exception for details on the failure. Visit the ACME certificate authority website directly to review the terms of service.
+
+# "Error creating account on ACME server."
+CWPKI2018E=CWPKI2018E: The user account could not be created on the ACME certificate authority at URI {0}. The error is {1}.
+CWPKI2018E.explanation=The user account creation request failed.
+CWPKI2018E.useraction=Review the provided exception for details on the failure.
+
+# "URL for ACME CA account: " + account.getLocation());
+CWPKI2019I=CWPKI2019I: The account URL provided by the ACME certificate authority at URI {0} is {1}.
+CWPKI2019I.explanation=The account was successfully created.
+CWPKI2019I.useraction=No action is required.
+
+#"Error reading domain key file."
+CWPKI2020E=CWPKI2020E: Failed to read the domain key file for the ACME certificate authority account at URI {0}. The file location is {1}. The error is {2}.
+CWPKI2020E.explanation=The domain key file for the certificate authority account could not be opened.
+CWPKI2020E.useraction=Review the provided exception for details on the failure. Verify the file location is correct and has the correct file permissions.
+
+#"Error reading account key file."
+CWPKI2021E=CWPKI2021E: Failed to read the account key file for the ACME certificate authority account at URI {0}. The file location is {1}. The error is {2}.
+CWPKI2021E.explanation=The account key file for the certificate authority account could not be opened.
+CWPKI2021E.useraction=Review the provided exception for details on the failure. Verify the file location is correct and has the correct file permissions.
+
+#"Error writing domain key file."
+CWPKI2022E=CWPKI2022E: Failed to write to the domain key file for the ACME certificate authority account at URI {0}. The file location is {1}. The error is {2}.
+CWPKI2022E.explanation=Could not write to the domain key file for the certificate authority account.
+CWPKI2022E.useraction=Review the provided exception for details on the failure. Verify the file location is correct and has the correct file permissions.
+
+#"Error writing account key file."
+CWPKI2023E=CWPKI2023E: Failed to write the account key file for the ACME certificate authority account at URI {0}. The file location is {1}. The error is {2}.
+CWPKI2023E.explanation=Could not write to the account key file for the certificate authority account.
+CWPKI2023E.useraction=Review the provided exception for details on the failure. Verify the file location is correct and has the correct file permissions.
+
+# "Error revoking certificate."
+CWPKI2024E=CWPKI2024E: Failed to revoke the requested certificate for the ACME certificate authority at URI {0}. The certificate that failed to revoke has serial number {1}. The error is {2}.
```

Comment:
The certificate that failed to revoke has serial number {1}.

is the certificate revoking or being revoked?